online-inference/vllm/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM ghcr.io/coreweave/ml-containers/torch-extras:c1bf355-nccl-cuda12.2.2-ubuntu22.04-nccl2.19.3-1-torch2.2.2-vision0.17.2-audio2.2.2
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -qq update && \
+    apt-get -qq install --no-install-recommends -y git curl && \
```

`libsodium23` in case user wants to use encryption.

```suggestion
    apt-get -qq install --no-install-recommends -y git curl libsodium23 && \
```